recipes/stringmlst/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: stringmlst
+  version: "0.3.6.1"
+
+source:
+  url: https://github.com/jordanlab/stringMLST/archive/0.3.6.1.tar.gz
+  md5: 13d8eb71e17abe0294989fef05071776
+
+build:
+  number: 0
+  skip: True  # [osx]
+    
+
+requirements:
+  build:
+    - pyfaidx ==0.4.8.1    
+
+  run:
+    - bwa ==0.7.13
+    - bedtools ==2.24
+    - samtools ==1.3
+    - pyfaidx ==0.4.8.1
+    - python
+
+test:
+  commands:
+    - stringMLST.py  -v
+
+about:
+  home: https://github.com/jordanlab/stringMLST
+  license: Custom
```

Comment:
The current license in the repo is a custom one written by the authors. I poke them and they decided to take https://creativecommons.org/licenses/by-nc-sa/4.0/ ! So that is awesome but they indicate that the changes will only take effect April 21.... So will not merge till then.